person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative './nameable'
+
```

Comment:
```person = Person.new(22, 'maximilianus')
   person.correct_name 
   capitalizedPerson = CapitalizeDecorator.new(person)
   capitalizedPerson.correct_name
   capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
   capitalizedTrimmedPerson.correct_name```

[OPTIONAL]
- Kindly import classes needed to run the code in person.rb so there are no errors in  the terminal